### DISCUSSION TITLE
Who's managing the Ko-fi?

### DISCUSSION THREAD

**Discussion by asylumexp** [community-scripts/ProxmoxVE#1123] [Q&A] [answered]
I noticed that a new [ko-fi was made for community scripts ](https://ko-fi.com/community_scripts), who's currently managing this and where is the money from the Kofi going? I can't find any mention of a kofi in discussions or issues. Maybe I missed something?

**Comment by Mellowlynx** [comment, marked as answer]
From the repository README.md:

We offer two donation options to help maintain and grow this project:

- Ko-Fi for tteck: [Donate to tteck's wife](https://ko-fi.com/proxmoxhelperscripts) - All donations will go directly to Angie, wife of the founder of this project https://github.com/community-scripts/ProxmoxVE/discussions/237.
- Ko-Fi for Community Edition: [Donate to this project](https://ko-fi.com/community_scripts) - All funds will go towards script maintenance infrastructure and server costs. **Our most immediate need is funding testing infrastructure**. Your contributions help keep the project running. 
To honor tteck's legacy this project will also raise money for charity (cancer research, hospice care). Of the money donated to this project, 30% will be donated to charity. Income, expenditure and charitable donations will be disclosed annually in a transparent manner.

**Comment by MickLesk** [comment]
Is that enough input and/or transparency for you?
I think most of it went to tteck, our site is a bit “hidden” at the moment :-) I think we will let tteck have his site for a bit, then it will probably be removed.
